### PR TITLE
Enable "maintenance mode is on" test in "operator_test.go"

### PR DESCRIPTION
**Pull request by sbodagala**
# Description

- Get "maintenance mode is on" test to succeed and enable it.

## Type of change

*Please select one of the options below.*

- Bug fix (non-breaking change which fixes an issue); testing related

## Discussion

*Are there any design details that you would like to discuss further?*

No.

## Testing

*Please describe the tests that you ran to verify your changes. Unit tests?
Manual testing?*

Manual testing.

*Do we need to perform additional testing once this is merged, or perform in a larger testing environment?*

Yes, nightly regression testing.

## Documentation

*Did you update relevant documentation within this repository?*

N/A

*If this change is adding new functionality, do we need to describe it in our user manual?*

N/A

*If this change is adding or removing subreconcilers, have we updated the core technical design doc to reflect that?*

N/A

*If this change is adding new safety checks or new potential failure modes, have we documented and how to debug potential issues?*

N/A

## Follow-up

*Are there any follow-up issues that we should pursue in the future?*

No.

*Does this introduce new defaults that we should re-evaluate in the future?*

No.

### CODE REVIEW

File: e2e/test_operator/operator_test.go [hunk 1]
```diff
@@ -98,6 +98,7 @@ var _ = BeforeSuite(func() {
 			chaosmesh.OneMode,
 		)
 	}
+
```
Comment (johscheuer) [inline e2e/test_operator/operator_test.go:101]: Can we remove this newline?

**Review by johscheuer — APPROVED** [review]
LGTM 👍

File: e2e/test_operator/operator_test.go (hunk 1 above)
Comment (sbodagala) [reply e2e/test_operator/operator_test.go]: Done.

**Review by johscheuer — APPROVED** [review]
Thanks!

**Pull request merged**